FrontEnd/src/Categories.jsx

```diff
@@ -9,3 +19,28 @@ export const CATEGORIES = [
   "Pet Sitting",
   "Chef",
 ];
+
+const fetchCategoryNames = async() =>{
+  const categoryDocRef = collection(database, POST_CATEGORIES_FOLDER_NAME);
+
+  try{
+    const querySnapshot = await getDocs(categoryDocRef);
+    const newCategories = [...CATEGORIES]
+    if (querySnapshot.empty) {
+      return CATEGORIES;
+    }
+
+    querySnapshot.forEach((doc) => {
```

Comment:
You can use .filter() or .map() function here

Reply:
Is .map a more efficient method for looping through the list in this case?

Reply:
It's mostly for readability reasons.